srcpkgs/LACT/patches/malloc_trim.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/lact-daemon/src/server/handler.rs b/lact-daemon/src/server/handler.rs
+index 93dc86c..0ef85fd 100644
+--- a/lact-daemon/src/server/handler.rs
++++ b/lact-daemon/src/server/handler.rs
+@@ -107,12 +107,6 @@ impl<'a> Handler {
+         };
+         handler.apply_current_config().await;
+ 
+-        // Eagerly release memory
+-        // `load_controllers` allocates and deallocates the entire PCI ID database,
+-        // this tells the os to release it right away, lowering measured memory usage (the actual usage is low regardless as it was already deallocated)
+-        unsafe {
+-            libc::malloc_trim(0);
+-        }
```

Comment:
musl does not have `malloc_trim`

Reply:
It might be worth considering applying this patch only on musl and leaving the code intact on glibc.